sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -405,6 +405,78 @@ class StatisticsSuite extends QueryTest with TestHiveSingleton with SQLTestUtils
     }
   }
 
+  test("check column statistics for case sensitive columns") {
+    val tableName = "tbl"
+    // scalastyle:off
+    // non ascii characters are not allowed in the source code, so we disable the scalastyle.
+    val columnGroups: Seq[(String, String)] = Seq(("c1", "C1"), ("列c", "列C"))
+    // scalastyle:on
+    columnGroups.foreach { case (column1, column2) =>
```

Comment:
Could you create a separate function for the following checking logics? Then, you can have two test cases without duplicate codes.